pkg/ddl/ddl_api.go

```diff
@@ -2829,31 +2829,16 @@ func (d *ddl) createTableWithInfoJob(
 		args = append(args, ctx.GetSessionVars().ForeignKeyChecks)
 	}
 
-	involvingSchemaInfo := make([]model.InvolvingSchemaInfo, 0, len(tbInfo.ForeignKeys)+1)
```

Comment:
why it's removed? and below FK for multi schema change

Reply:
I want to allow CREATE TABLE t1 FOREIGN KEY t0.c0 and CREATE TABLE t2 FOREIGN KEY t0.c0 running concurrently in this PR. See below.

https://github.com/pingcap/tidb/pull/54003#discussion_r1644067821

Reply:
Oh, CI is failed because of this...

TestTruncateOrDropTableWithForeignKeyReferred2